setup.json

```diff
@@ -19,5 +19,25 @@
 		"prospector==1.3.0",
 		"pylint==2.5.2"
         ]
-    }
+    },
+    "install_requires": [
+        "aiidalab-widgets-base~=1.0.0b7",
+        "arrow~=0.15.5",
+        "binaryornot~=0.4.4",
+        "bokeh~=2.1.1",
+        "bqplot~=0.12.14",
+        "cookiecutter~=1.6.0",
+        "dulwich~=0.19.15",
+        "ipywidgets<8.0.0",
+        "lxml~=4.5.2",
+        "Markdown~=3.2.2",
+        "notebook~=5.7.10",
+        "packaging~=20.1",
+        "pip~=20.2",
+        "pythreejs~=2.2.0",
+        "pysmiles~=1.0.1",
+        "requests-cache~=0.5.2",
+        "watchdog~=0.10.2",
+        "widget_periodictable~=2.1.2"
```

Comment:
Most of those should be app dependencies, like `aiidalab-widgets-base`. Could you have another look at this and provide some justification for the dependencies you would like to keep?

Reply:
sure:
`aiidalab-widgets-base` base widgets for aiidalab.

`arrow` - no idea

`binaryornot` - no idea

`bokeh` and `bqpot` widely used plotting software + requires extension installation.

`cookiecutter` required to start new apps development, ok to move to the docker container.

`dulwich` required by aiidalab

`ipywidgets` generally needed

`lxml` can be deleted

`Markdown` can be deleted

`notebook` is generally needed, but I can move it to the docker container.

`packaging` is also generally needed, but it can be installed afterwards. Ok to remove.

`pip` is generally needed, but I can move it to the docker container.

`pythreejs` not sure why is it needed.

`pysmiles` can be deleted.

`requests-cache` is required by aiidalab

`watchdog` is required by aiidalab

`widget_periodictable` requires activation.

Reply:
only 3 dependencies left 😎